stdlib/IterativeEigenSolvers/src/IterativeEigenSolvers.jl

```diff
@@ -73,6 +75,13 @@ julia> λ
 2-element Array{Float64,1}:
  4.0
  3.0
+
+julia> λ, ϕ = eigs(A, nev = 2, which=:SM);
+
+julia> λ
+2-element Array{Float64,1}:
+ 1.0000000000000002
```

Comment:
These might be problematic if we turn on doctest on CI since it does not always return the same thing. But the existing example is just as problematic so we can fix that later if it turns out to be a real problem.
```julia
julia> λ, ϕ = eigs(A, nev = 2, which=:SM); λ
2-element Array{Float64,1}:
 1.0000000000000002
 2.0               

julia> λ, ϕ = eigs(A, nev = 2, which=:SM); λ
2-element Array{Float64,1}:
 1.0               
 2.0000000000000004

julia> λ, ϕ = eigs(A, nev = 2, which=:SM); λ
2-element Array{Float64,1}:
 1.0000000000000002
 2.0               

julia> λ, ϕ = eigs(A, nev = 2, which=:SM); λ
2-element Array{Float64,1}:
 1.0000000000000002
 1.9999999999999996
```